lib/puppet/http/service/report.rb

```diff
@@ -0,0 +1,43 @@
+class Puppet::HTTP::Service::Report < Puppet::HTTP::Service
+  API = '/puppet/v3'.freeze
+
+  # copied from lib/puppet/indirector/rest.rb
```

Comment:
I don't think this is a useful comment, but I wasn't sure why these formats are specifically excluded